engine/apps/mattermost/serializers.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework import serializers
+
+from apps.mattermost.client import MattermostClient
+from apps.mattermost.exceptions import MattermostAPIException, MattermostAPITokenInvalid
+from apps.mattermost.models import MattermostChannel
+from common.api_helpers.utils import CurrentOrganizationDefault
+
+
+class MattermostChannelSerializer(serializers.ModelSerializer):
+    id = serializers.CharField(read_only=True, source="public_primary_key")
+    organization = serializers.HiddenField(default=CurrentOrganizationDefault())
+
+    class Meta:
+        model = MattermostChannel
+        fields = [
+            "id",
+            "organization",
+            "mattermost_team_id",
+            "channel_id",
+            "channel_name",
+            "display_name",
+            "is_default_channel",
+        ]
+
+    def create(self, validated_data):
+        return MattermostChannel.objects.create(**validated_data)
+
+    def to_representation(self, instance):
+        ret = super().to_representation(instance)
+        del ret["mattermost_team_id"]
```

Comment:
Why is this needed?

Reply:
I don't see a need for us to expose the `mattermost_team_id` to UI hence wanted to remove that from payload. Modified the serializers Meta class and included `write_only` flag for neater representation.
Do you feel we should send this in payload ?